barretenberg/cpp/src/barretenberg/vm/avm/trace/public_inputs.hpp

```diff
@@ -18,8 +18,8 @@ struct Gas {
 inline void read(uint8_t const*& it, Gas& gas)
 {
     using serialize::read;
-    read(it, gas.l2_gas);
     read(it, gas.da_gas);
+    read(it, gas.l2_gas);
```

Comment:
Is this intentional? Or is this accidentally reverting some change that was made in master?

Reply:
it was actually the wrong way around originally!